### ISSUE TITLE
APKs using deprecated MD5 signature

### ISSUE THREAD

**Issue by IzzySoft** [pelya/screen-dimmer-pixel-filter#15]
Just a note: your APKs are still signed using MD5, which is deprecated.

    ERROR: JAR signer PELYA.RSA: Failed to verify JAR signature META-INF/PELYA.RSA against META-INF/PELYA.SF: java.security.SignatureException: Algorithm constraints check failed on disabled algorithm: MD5.

As you know, deprecated soon might mean unsupported – so you might wish to act on this :wink:

**Comment by IzzySoft** [comment]
@pelya Just to inform you that at least one other project was able to solve this issue  (see [here](https://github.com/AdguardTeam/AdguardForAndroid/issues/1575#issuecomment-350717523)). Their solution might apply to your project as well. Quoting:

> The solution was clearly one of the:
>
>* Transition from Maven to Gradle build system.
>* Update of Android SDK and build tools.
>
> But no change in certificates and keys was made.

Worth a look I'd say – and as you're already using Gradle, it might be the other bullet point (which is what folks at AdGuard think as well). Same for issue 107 at [android-keyboard-gadget](https://github.com/pelya/android-keyboard-gadget/issues/107).

**Comment by IzzySoft** [comment]
@pelya is this project abandoned?

**Comment by pelya** [comment]
Yes. Android 8 and newer disallows drawing over notification panel and
system navigation bar, so the app cannot work properly on it. I keep it
published only because someone asked it.

**Comment by IzzySoft** [comment]
Thanks for the sad news, @pelya – but at least the status is clear now. May I suggest you mention that in the README? Optionally, you could also archive this project via its settings.

**Comment by pelya** [comment]
Updating the readme done. I'll archive the repo.
